packages/block-editor/src/components/block-alignment-matrix-toolbar/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { DOWN } from '@wordpress/keycodes';
+import {
+	Button,
+	Dropdown,
+	ToolbarGroup,
+	__experimentalAlignmentMatrixControl as AlignmentMatrixControl,
```

Comment:
I know we have this tendency to make a lot of things experimental but I'd like to ask what level of confidence we have over the naming and the props (API) of this component. If it's a high level, maybe we can just make it stable?

Reply:
> what level of confidence we have over the naming and the props (API) of this component

I wasn't sure of that distinction either. I've been being conservative, and defaulting to using the `__experimental` prefix.

At being said... My thinking is that I'm currently feeling quite confident. In that case...

* Merge
* Wait a couple of weeks to see if any feedback/bugs pop up
* If nothing, then, safe to remove experimental flag

Reply:
Agreed, I think we should possibly apply this to the _Group_ block once merged and see if it works fine before considering removing the _experimental prefix.